### ISSUE TITLE
Zathura is missing necessary plugins 

### ISSUE THREAD

**Issue by IndianaRoy** [clearlinux/distribution#2277]
Official package name: zathura

Download URL of latest release: 
[https://pwmt.org/projects/zathura-djvu/](url)
[https://pwmt.org/projects/zathura-ps/](url)
[https://pwmt.org/projects/zathura-cb/](url)

Description:
zathura only comes with the zathura-pdf-poppler package, it's missing the plug-ins for DjVu, PostScript and comic book files

**Comment by IndianaRoy** [comment]
I can't make it open PDF files neither, does the another bundle includes [girara](https://pwmt.org/projects/girara/)?

**Comment by phmccarty** [comment]
zathura depends on girara already... From the list of runtime requirements, I see `libgirara-gtk3.so.3()(64bit)`, which is provided by girara package.

**Comment by phmccarty** [comment]
I just checked the current `zathura` [bundle contents](https://cdn.download.clearlinux.org/update/33900/Manifest.zathura), and I see the girara library (plus soname symlink), the zathura binary, the libpdf-poppler plugin, and the remaining files from the zathura package.

I can't test a live system at the moment, but I can try to reproduce your issue tomorrow. If you have any more details about the problem, feel free to post them.

**Comment by IndianaRoy** [comment]
You're right I just check and have `Component girara has version 0.3.5` and `Component zathura has version 0.4.7`
This is what I get when I run zathura:
`error: could not open plugin directory: /usr/local/lib64/zathura`
I just checked, for some reason the directory doesn't have the plugins
`ls -a /usr/local/lib64`
`libgirara-gtk3.so    libgirara-gtk3.so.3.1
libgirara-gtk3.so.3  pkgconfig`

**Comment by IndianaRoy** [comment]
Thanks @phmccarty I just made the directory and copy the `libpdf-poppler.so` from `/usr/lib64/zathura` to `/usr/local/lib64/zathura` and it's working now.

All that's left now is for the `zathura` bundle to include the other plug ins

**Comment by IndianaRoy** [comment]
Any chance to add [mupdf](https://pwmt.org/projects/zathura-pdf-mupdf/)?
To be able to read epub files

**Comment by IndianaRoy** [comment]
When trying to build the plugins I got the following error
`Run-time dependency zathura found: NO (tried pkgconfig and cmake)`

**Comment by lebensterben** [comment]
> When trying to build the plugins I got the following error
> 
> `Run-time dependency zathura found: NO (tried pkgconfig and cmake)`

Because the pkg-config file is not installed. You can modify one from its template.
